### PR TITLE
teamcity-server

### PR DESCRIPTION
**Pull request by skyguy94**
This PR contains a chocolatey package for teamcity server.

### CODE REVIEW

File: teamcity-server/teamcity.nuspec [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+`choco install teamcity-server -params "username=*user* password=*password* domain=*domain*"` to specify a domain user account for the TeamCity service. This defaults to the SYSTEM account.
+
+`choco install teamcity-server -params "unzipLocation=*path*"` to specify the root directory for the TeamCity service. This defaults to `C:\`.
+Note: A TeamCity folder is always created, so the actual install path is `C:\TeamCity` for the default case or *path*\TeamCity with a custom option.
+</description>
+
+    <summary>This package installs the continuous integration server without a build agent. Windows 32 and 64 bit. Default install location: `C:\TeamCity.`</summary>
+    <releaseNotes>https://confluence.jetbrains.com/display/TW/TeamCity+9.0.3+%28build+32334%29+Release+Notes</releaseNotes>
+    <copyright>JetBrains s.r.o.</copyright>
+    <tags>>Testing Build Coding Compilation Continuous Integration CI jetbrains admin</tags>
```
Comment (derigel23) [inline teamcity-server/teamcity.nuspec]: Mistyping?

Comment (skyguy94) [inline teamcity-server/teamcity.nuspec]: I don't see it?

Comment (derigel23) [inline teamcity-server/teamcity.nuspec]: `<tags>>`

Comment (skyguy94) [inline teamcity-server/teamcity.nuspec]: Fixed. TY!

**Comment by skyguy94** [comment]
Poke...any interest in this?

**Comment by skyguy94** [comment]
Ok, given the silence, I'll delete the PR.

**Pull request closed without merging**